examples/animation/animated_transform.rs

```diff
@@ -0,0 +1,140 @@
+use std::f32::consts::{FRAC_PI_2, PI};
+
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 1.0,
+        })
+        .add_startup_system(setup)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    mut animations: ResMut<Assets<AnimationClip>>,
+) {
+    // Camera
+    commands.spawn_bundle(PerspectiveCameraBundle {
+        transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
+        ..default()
+    });
+
+    // Animation use name to target entities.
+    let planet = Name::new("planet".to_string());
+    let orbit_controller = Name::new("orbit_controller".to_string());
+    let satellite = Name::new("satellite".to_string());
```

Comment:
```suggestion
    let planet = Name::new("planet");
    let orbit_controller = Name::new("orbit_controller");
    let satellite = Name::new("satellite");
```
Minor nit. Looking good!